app.json

```diff
@@ -2,6 +2,7 @@
   "name":"vets-website",
   "scripts":{},
   "env":{
+    "INSTALL_HOOKS": "no"
```

Comment:
As far as I was aware `app.json` was only applicable to the Heroku environment, so I intended this change to set the value to "no" only for Heroku. Is it used somewhere else?